src/credentials/tests/TestChipCert.cpp

```diff
@@ -1345,6 +1345,23 @@ static void TestChipCert_GenerateRootCert(nlTestSuite * inSuite, void * inContex
     NL_TEST_ASSERT(inSuite, ConvertX509CertToChipCert(signed_cert_span2, outCert) == CHIP_NO_ERROR);
     NL_TEST_ASSERT(inSuite, DecodeChipCert(outCert, certData) == CHIP_NO_ERROR);
 
+    // Test with no defined notAfter time.
```

Comment:
We should also add a test case to `TestChipCert_ChipToX509` and `TestChipCert_X509ToChip` that tests a cert with such a NotAfter time. Currently all the test vector certs seem to use the exact same NotBefore/NotAfter range...

Reply:
Done.  That test passes even without this PR, but it's good to have it.